x-pack/plugins/beats_management/public/components/table/table_type_configs.tsx

```diff
@@ -282,7 +282,7 @@ export const TagsTableType: TableType = {
     actions: [
       {
         name: i18n.translate('xpack.beatsManagement.tagsTable.removeSelectedLabel', {
-          defaultMessage: 'Remove Selected',
+          defaultMessage: 'Remove selected',
```

Comment:
I'm not sure. I think this was one of the strings that wasn't being used by the UI. I edited it anyhow, but couldn't add more context because I could not confirm it in the UI. Maybe @mattapperson can provide more context.

Reply:
@mattapperson Still waiting for your input here.

Reply:
In this case its an action to remove selected tags on the tags table. I did not include the word tags here because it seemed redundant as the page and table only display tags and they are the only selectable things

Reply:
I think it's an EUI standard though...to indicate what you are removing. @gchaps Is that correct?

Reply:
@mattapperson Can you tell me where this is used (maybe provide a screen capture)? Note that I did edit the text on the button on the Configuration Tags tab to say "Delete selected tags" instead of "Remove selected" (see the [edit](https://github.com/elastic/kibana/pull/30338/files#diff-60fc387f014e29061077240572e95e51R334) here:

![image](https://user-images.githubusercontent.com/14206422/54164174-e39afd00-4418-11e9-998f-891286f93171.png)

But I cannot see where the string on line 285 is used. Again, I hesitate to change strings that I cannot directly validate in the UI.

Reply:
Sorry, you are correct that this label is no longer used. My mistake

Reply:
@mattapperson Should I delete it, or leave it?